pallets/gear/src/tests.rs

```diff
@@ -630,7 +635,7 @@ fn delayed_send_user_message_with_reservation() {
             &(delay + 3),
             &ScheduledTask::SendUserMessage {
                 message_id: delayed_id,
-                to_mailbox: false
+                to_mailbox: true
```

Comment:
did we cover all possible cases with tests? recheck it pls

by destination type:
- user (to mailbox, not to mailbox cases)
- program

by sending type:
- common one
- from reservation

Reply:
All this cases are covered. I implemented the following tests:
 - delayed_send_user_message_payment
 - delayed_send_user_message_with_reservation
 - delayed_send_program_message_payment
 - delayed_send_program_message_with_reservation
 - delayed_send_program_message_with_low_reservation